lib/bootloader_setup.pm

```diff
@@ -35,17 +36,32 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+# prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
+sub stop_grub_timeout {
```

Comment:
The fact that @okurz decided to create this function showing that we have some common logic spread in several places. Just in doubt that only  "send_key 'up' " line can be included here. 
1. I noticed that in all cases  (except reboot_and_install.pm) before doing send_key we asserting for 'grub2' needle. Probably we can include this check inside this function to make it "longer than one line" ? 
2. in comment to function you saying "better to use esc but to be more generic let's use up" so if you decided to create a function why not split this cases ? based on env. variables detect when to use esc and when up ?
3. I think we need to investigate problem in this direction - what else is in common in places where we try to call this function ?